### ISSUE TITLE
Windows Defender now flagging BrowseRouter as a Trojan

### ISSUE THREAD

**Issue by M3ntalBadg3r** [nref/BrowseRouter#30]
Hi,
This morning, Windows Defender flagged and deleted BrowseRouter as a trojan, specifically "Trojan:Win32/Wacatac.B!ml"

![image](https://github.com/user-attachments/assets/5afe31d3-05ee-4fb7-a88d-1ecdc23bd64d)

Not sure whats changed

**Comment by nref** [comment]
Thanks for reporting this. I can't do anything about it. I spent about 30 minutes uploading various versions to VirusTotal. I got it down from from 15 to 4 out of 74 scanners flagging it as malware. I can't spend any more time on it. 

Scanners are known to overzealously flag false positives at random. A search for the specific one above shows that is sometimes the case for `Wacatac.B!ml`.

BrowseRouter is clean. You can read the source and build it yourself. It does elevate to admin when you use the `--register` command, and it reads and writes registry keys. Those could register on a scanner's heuristics.

Moreover, Microsoft has an interest in preventing alternative browsers.

**Comment by nref** [comment]
I have just released [version 0.7.2](https://github.com/nref/BrowseRouter/releases/tag/0.7.2.0) which may address some scanners.

On VirusTotal, this brings the number of flagged vendors down from [15 to 4](https://www.virustotal.com/gui/file/c37c835cf52b1ae35003849dca1066f05dee3b21749b714f9fc9392b9ca8f881?nocache=1).

![image](https://github.com/user-attachments/assets/cd4a8294-4523-445f-b3b5-c2d156f35b0d)

I cannot do much about the remainder. BrowseRouter contains no malware, but these scanners' heuristics think some part of it resembles malware. Keep in mind BrowseRouter makes no network connections. That would make it fairly useless to a bad actor.

I have also cryptographically signed the executable, which proves I created them and they are not modified by some middleman.

![image](https://github.com/user-attachments/assets/c3102f46-3cf2-436f-9b77-45deb5c2e75f)
